src/api/types/ApiResponses.ts

```diff
@@ -46,6 +46,7 @@ export type PaymentRequest = {
   cardStripePaymentIntentID?: string;
   cardStripePaymentIntentSecret?: string;
   addresses: PaymentRequestAddress[];
+  events: PaymentRequestEvent[];
```

Comment:
It seems like you've added `events`. I presume you intended to use them to enable or disable the delete button, but you ended up using `attempts` instead. Was that intentional?

And while we're on the subject, what are the differences between them?

Reply:
I originally intended to use `events`, but was informed that `attempts` was a better choice since it's practically the same as `events` and it was readily available to use in the code.